README.md

```diff
@@ -40,6 +40,10 @@ UiPath Pack is available in standard jobs and pipelines, and lets you package an
 | Choose versioning method      | UiPath packages are versioned. With UiPath pack you can choose between 3 different options: (1) Auto generate (2) Define custom version (3) Use the current version set in the project. |
 | Project(s) path               | The location of the project(s) to be packaged. It can be a direct path to a project.json file or a directory with one or multiple projects. In the latter case, each level one project is packaged individually. |
 | Output folder                 | Path to a folder, where the created package should be placed. |
+| Output type                   | The output type of the project(s). There are 5 options for the project(s) type: *(1)* Output type of the project *(2)* Pack a process project *(3)* Pack a library project *(4)* Pack a tests project *(5)* Pack an objects project |
+| Orchestrator address          | The address of the Orchestrator instance onto which the package(s) will be deployed. |
```

Comment:
This should rather be `The address of the Orchestrator instance from which library dependencies should be restored.`, since we don't deploy anything over here. Same goes for tenant.

Let's also document the `useOrchestrator` checkbox, noting that you only need to add this for packaging library projects which uses dependencies that must be restored from the specified Orchestrator instance.